packages/cli/src/types.ts

```diff
@@ -16,7 +16,7 @@ export interface JSONObject {
 }
 
 export interface AuthConfig {
-  token: string;
+  token?: string;
```

Comment:
VS Code was complaining (though the version of TypeScript we use to build doesn't seem to mind).

Reply:
@TooTallNate You can tell VS Code to use the same version as the build by selecting the version in the bottom right of VS Code > Select TypeScript Version > Use Workspace Version.

https://stackoverflow.com/a/59076553/266535

Reply:
Oh, nice TIL.